kostyor/conf.py

```diff
@@ -0,0 +1,13 @@
+from oslo_config import cfg
+config_files = ['conf.ini']
```

Comment:
Hm.. I think we don't need the `conf.ini` if we're going to use `oslo.cfg`. How about follow OpenStack approach and set defaults in python code within `default_opts` below?